README.md

```diff
@@ -69,12 +69,12 @@ buildscript {
         }
     }  
     dependencies {  
-        classpath "gradle.plugin.com.polidea.cockpit:CockpitPlugin:1.0.3"  
+        classpath "gradle.plugin.com.polidea.cockpit:CockpitPlugin:1.1.1-SNAPSHOT"  
```

Comment:
There are some more changes to be made in readme, let's remember about them. ;)

Reply:
True